string sweep_detection
float64 distance_of_sweep
